src/pages/overview/document-generation-api/inlineimages.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Document Services APIs | Document Generation API | Inline Images
+---
+# Inline Images
+
+Add inline images in the document through json data using the <img\> tag.
+
+## How It Works
+
+You can provide the input image url by making use of html img tag in json. This is one such sample json.
```

Comment:
Instead of this 'This is one such sample json.', use 'Below is the example json data.'

Reply:
If we don't support base64 inline data type images (not using the right term, but you know what I mean), please clarify that in the docs.

Reply:
Made the necessary changes, also added the note for the limitation.